### PR TITLE
872 about us page

### PR DESCRIPTION
**Pull request by sjpaige**
About Us page is completed. Any thoughts on things that can be made better are greatly appreciated. I need to update the footer to the final but I will do that in its own branch since it is independent.

### CODE REVIEW

File: src/data/teamData.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Seamus_Brugh from "../Assets/team/Seamus Brugh.png";
```
Comment (stuttskl) [inline src/data/teamData.ts:1]: Do we want to store team photos in an S3 bucket? Or in our project repo?

File: src/Pages/AboutPage.jsx [hunk 2]
```diff
@@ -3,92 +3,297 @@ import {
...
+                </Typography>
+              }
+            />
+          </List>
+        </Container>
+      </Box>
+      <Box className={classes.problemRoot}>
+        <Container maxWidth="lg">
+          <Grid container>
+            <Grid className={classes.theProblemRootleft} item sm={12} md={6}>
```
Comment (stuttskl) [inline src/Pages/AboutPage.jsx:187]: Could we decrease the top padding _slightly_ here? Here's an image for reference

File: src/data/teamData.ts (hunk 1 above)
Comment (sjpaige) [reply src/data/teamData.ts:1]: We can store them outside the repo. I was just getting the basic component built.

File: src/Pages/AboutPage.jsx (hunk 2 above)
Comment (sjpaige) [reply src/Pages/AboutPage.jsx:187]: Not sure the reference uploaded properly.

Comment (sjpaige) [reply src/Pages/AboutPage.jsx:189]: Let me know what you think of the newer padding.

File: src/data/teamData.ts (hunk 1 above)
Comment (sjpaige) [reply src/data/teamData.ts:1]: Would you like me to setup the S3 or is that something that Dylan wants to do, or you?

Comment (stuttskl) [reply src/data/teamData.ts:1]: I created an S3 bucket on the AWS account Dylan set up, but need to read more about pricing!

File: src/Pages/AboutPage.jsx (hunk 2 above)
Comment (stuttskl) [reply src/Pages/AboutPage.jsx:189]: Looks great! 😄

**Review by stuttskl — APPROVED** [review]

File: src/data/teamData.ts (hunk 1 above)
Comment (sjpaige) [reply src/data/teamData.ts:1]: Should I merge this one and create another PR to add the images and modify the team card component when we decide the final option? or hold of until we have it setup?

**Pull request merged**